src/analyservariable.cpp

```diff
@@ -16,9 +16,12 @@ limitations under the License.
 
 #include "libcellml/analyservariable.h"
 
+#include "libcellml/variable.h"
+
```

Comment:
This include should be below the system include.

Reply:
f24e786c